lib/shanty/plugin.rb

```diff
@@ -13,6 +13,10 @@ def self.inherited(plugin_class)
       @plugins << plugin_class.new
     end
 
+    def self.plugins
+      (@plugins || []).map(&:name)
```

Comment:
I'm not sure I'd do the `.map(&:name)` in here - I'd leave that out for the callee to do if they want.